### PR TITLE
fixed ValueError for AF_INET6 sockets

### PR DESCRIPTION
**Pull request by ffix**
## What do these changes do?

There is an issue if you pass a preexisting INET6 socket to the aiohttp.web.run_app function.
```
Traceback (most recent call last):
  File "test_aiohttp.py", line 26, in <module>
    web.run_app(app, sock=sock1)
  File "/usr/lib/python3.6/site-packages/aiohttp/web.py", line 435, in run_app
    backlog=backlog)
  File "/usr/lib/python3.6/site-packages/aiohttp/web.py", line 407, in _make_server_creators
    host, port = sock.getsockname()
ValueError: too many values to unpack (expected 2)
```

## Are there changes in behavior for the user?

It's a bugfix.

## Related issue number

There is no tracked issue.

## Checklist

- [x] I think the code is well written
- [x] Unit tests for the changes exist
- [ ] Documentation reflects the changes
- [x] If you provide code modification, please add yourself to `CONTRIBUTORS.txt`
  * The format is &lt;Name&gt; &lt;Surname&gt;.
  * Please keep alphabetical order, the file is sorted by names. 
- [x] Add a new news fragment into the `changes` folder
  * name it `<issue_id>.<type>` for example (588.bug)
  * if you don't have an `issue_id` change it to the pr id after creating the pr
  * ensure type is one of the following:
    * `.feature`: Signifying a new feature.
    * `.bugfix`: Signifying a bug fix.
    * `.doc`: Signifying a documentation improvement.
    * `.removal`: Signifying a deprecation or removal of public API.
    * `.misc`: A ticket has been closed, but it is not of interest to users.
  * Make sure to use full sentences with correct case and punctuation, for example: "Fix issue with non-ascii contents in doctest text files."

### CODE REVIEW

File: tests/test_run_app.py [hunk 1]
```diff
@@ -42,6 +42,16 @@
 )
 del _has_unix_domain_socks, _abstract_path_failed
 
+has_ipv6 = socket.has_ipv6
```
Comment (asvetlov) [inline tests/test_run_app.py]: `has_ipv6` should be upper-cased: it is a contant

Comment (ffix) [reply tests/test_run_app.py]: I've copied it from another test [module](https://github.com/aio-libs/aiohttp/blob/master/tests/test_http_stream_writer.py#L9) as is.

Comment (asvetlov) [reply tests/test_run_app.py]: But please make it upper cased anyway

Comment (ffix) [reply tests/test_run_app.py]: OK, done.

**Comment by asvetlov** [comment]
thanks!

**Pull request merged**